frontend/src/components/CreateProjectDialog.tsx

```diff
@@ -1,94 +1,103 @@
-            const response = await fetch(`/api/v1/project/${projectID}`, {
-                method: 'POST',
-                headers: {
-                    'Content-Type': 'application/json',
+            const response = await fetch(
+                `/api/v1/project/${projectID}?name=${encodeURIComponent(projectName)}&description=${encodeURIComponent(description)}`,
+                {
+                    method: "POST",
                 },
-                body: JSON.stringify({
-                    name: projectName,
-                    description: description,
-                }),
-            });
-
+            );
             if (!response.ok) {
                 setError(ErrorType.ProjectCreationFailed);
-            } else {
-                closeDialog(); // Close dialog on successful creation
             }
-        } catch {
+        } catch (err) {
+            console.log(err);
             setError(ErrorType.ProjectCreationFailed);
         }
-        
-        closeDialog(); 
-    };
 
+        closeDialog();
+    };
 
     return (
         <>
-        {/* Render button if the dialog is not open */}
-        {!isDialogOpen && (
-            <button onClick={openDialog}>Create New Project</button>
-        )}
-
-        {/* Render dialog only if isDialogOpen is true */}
-        {isDialogOpen && (
-            <div>
-                <div className="dialog">
-                    <h2>Create New Project</h2>
-                    {error && <p className="error">{error}</p>}
-                    <form onSubmit={handleSubmit}>
-                        <input
-                            type="text"
-                            placeholder="Project Name"
-                            value={projectName}
-                            onChange={(e) => setProjectName(e.target.value)}
-                            required
-                        />
-                        <textarea
-                            placeholder="Description"
-                            value={description}
-                            onChange={(e) => setDescription(e.target.value)}
-                            required
-                        />
-                        <input
-                            type="text"
-                            placeholder="Project ID"
-                            value={projectID}
-                            onChange={(e) => setProjectID(e.target.value)}
-                            required
-                        />
-                        <button className="button" type="button" onClick={closeDialog}>Cancel</button>
-                        <button className="button" type="submit">Create Project</button>
-                    </form>
+            {/* Render button if the dialog is not open */}
+            {!isDialogOpen && (
+                <button onClick={openDialog}>Create New Project</button>
+            )}
+            {isDialogOpen && (
+                <div>
+                    <div className="dialog">
```

Comment:
Shouldn't this just be a `<dialog>`

Reply:
I didn't need both <div> but <dialog> does not work here.